### PR TITLE
[script.module.pyserial] 3.4.0

### PR DESCRIPTION
**Pull request by fredrik-eriksson**
### Description
The pyserial python module packaged for kodi. This package will be required for my projcontrol service addon.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by enen92** [comment]
@petterreinholdtsen @fredrik-eriksson we could not yet find time to review it. Please be patient.
I'll try to have a look tomorrow.

File: script.module.pyserial/LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (enen92) [inline script.module.pyserial/LICENSE]: License file should be the same the module provides: https://github.com/pyserial/pyserial/blob/master/LICENSE.txt

File: script.module.pyserial/addon.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyserial"
+       name="pyserial"
+       version="3.4.0"
+       provider-name="feffe">
```
Comment (enen92) [inline script.module.pyserial/addon.xml]: provider-name is the author of the module: Chris Liechti

File: script.module.pyserial/addon.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Python serial port access library</summary>
+    <description lang="en_GB">Packed for KODI from https://github.com/pyserial/pyserial</description>
+    <platform>all</platform>
+    <language></language>
+    <license>BSD-3-Clause</license>
+    <forum></forum>
+    <website>https://github.com/pyserial/pyserial</website>
+    <source>https://github.com/fredrik-eriksson/script.module.pyserial</source>
```
Comment (enen92) [inline script.module.pyserial/addon.xml]: I think the original module source repository fits better here but I'm fine as it is.

Comment (fredrik-eriksson) [reply script.module.pyserial/addon.xml]: Right, LICENSE and provider-name fixed, I'd like to keep the source as is as it contains the actual source code that is used to produce the plugin. If I mess up with the packaging and the source happens to differ from the original, I think the blame should go to me and not the original author :)

Comment (enen92) [reply script.module.pyserial/addon.xml]: I'm okay with that

**Pull request merged**